src/core/server/http/http_service.ts

```diff
@@ -85,7 +85,7 @@ export class HttpService implements CoreService {
           'Router will **not** be applied.'
       );
     } else {
-      this.log.info(`registering route handler for [${router.path}]`);
+      this.log.debug(`registering route handler for [${router.path}]`);
```

Comment:
__note:__ we're going to register lots of routes in the future and logging every one with `info` level is probably too much...